### ISSUE TITLE
list connected hosts

### ISSUE THREAD

**Issue by celesteking** [linux-nvme/nvme-cli#1191]
I'd like to see a list of connected hosts to a particular NVMeOF target.

Thanks.

**Comment by celesteking** [comment]
Yes, I understand, but where do I fill feature requests and questions regarding that? I've tried https://lists.infradead.org/pipermail/linux-nvme , but they don't give a shit about users.

**Comment by keithbusch** [comment]
It has nothing to do with not giving a shit. There is finite time with finite developers. This is open source, anyone can contribute, more developers are always appreciated.
